lib/ui/myplot/PlotList.dart

```diff
@@ -76,76 +74,101 @@ class _MyPlotState extends State<PlotList> {
       case LoadingStatus.ERROR:
-        return _buildErrorPage(context, viewModel, myPlotStyle); // TODO Check FARM-203
+        return _buildErrorPage(
+            context, viewModel, myPlotStyle); // TODO Check FARM-203
     }
   }
-}
 
-Widget _buildPage(BuildContext context, List<CropEntity> cropList,
-    PlotListStyle plotStyle, Function goToDetail){
-  final viewModel = buildPlotListViewModel();
-  return HeaderAndFooterListView.builder(
-      itemCount: cropList.length,
-      itemBuilder: (BuildContext context, int index) {
-        final itemViewModel = fromCropEntityToViewModel(cropList[index], goToDetail);
-        return PlotListItem().buildListItem(itemViewModel);
-  },
-  physics: ScrollPhysics(),
-  shrinkWrap: true,
-  header: _buildTitle(viewModel, plotStyle, context: context),
-  footer: RoundedButton.build(style: LargeRoundedButtonStyle() ,context: context, title: viewModel.buttonTitle));
-}
 
-Widget _buildTitle(PlotListViewModel viewModel, PlotListStyle myPlotStyle, {BuildContext context}){
-  return Container(
-    padding: myPlotStyle.titleEdgePadding,
-    child: Row(
-      mainAxisAlignment: MainAxisAlignment.spaceBetween,
-        children: <Widget>[
-          Column(
-            mainAxisAlignment: MainAxisAlignment.start,
+  Widget _buildPage(BuildContext context, List<CropEntity> cropList,
+      PlotListStyle plotStyle, Function goToDetail) {
+    final viewModel = buildPlotListViewModel();
+    return HeaderAndFooterListView.builder(
+        itemCount: cropList.length,
+        itemBuilder: (BuildContext context, int index) {
+          final itemViewModel = fromCropEntityToViewModel(
+              cropList[index], goToDetail);
+          return PlotListItem().buildListItem(itemViewModel);
+        },
+        physics: ScrollPhysics(),
+        shrinkWrap: true,
+        header: _buildTitle(viewModel, plotStyle, context: context),
+        footer: RoundedButton(viewModel: RoundedButtonViewModel(title: viewModel.buttonTitle, onTap: () => _showToast(context)),
+            style: RoundedButtonStyle.largeRoundedButtonStyle()));
+  }
+
+  Widget _buildTitle(PlotListViewModel viewModel, PlotListStyle myPlotStyle,
+      {BuildContext context}) {
+    return Container(
+        padding: myPlotStyle.titleEdgePadding,
+        child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
             children: <Widget>[
-              Text(
-                viewModel.title,
-                style: myPlotStyle.titleTextStyle,
-              )
+              Column(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  children: <Widget>[
+                    Text(
+                      viewModel.title,
+                      style: myPlotStyle.titleTextStyle,
+                    )
+                  ]
+              ),
+              RoundedButton(viewModel:
+                  RoundedButtonViewModel(icon: "assets/icons/profit_add.png", onTap: () => _showToast(context)),
```

Comment:
Don't think we should have this hardcoded text here with the asset name, move it to a constant